drivers/remote/driver.go

```diff
@@ -28,10 +28,25 @@ func newDriver(name string, client *plugins.Client) driverapi.Driver {
 // plugin is activated.
 func Init(dc driverapi.DriverCallback) error {
 	plugins.Handle(driverapi.NetworkPluginEndpointType, func(name string, client *plugins.Client) {
-		c := driverapi.Capability{
-			Scope: driverapi.GlobalScope,
+		// negotiate driver capability with client
+		d := newDriver(name, client)
+		var capResp api.GetCapabilityResponse
+		if err := d.(*driver).call("GetCapability", nil, &capResp); err != nil {
+			log.Errorf("error getting capability for %s due to %v", name, err)
```

Comment:
do you think it will be better to default the scope to be `GlobalScope` if the driver hasn't implemented `GetCapability` ?
That will atleast provide a compatibility with the existing drivers. WDYT ?